static/src/stylesheets/module/facia-garnett/_pillars.scss

```diff
@@ -183,7 +197,7 @@ $pillars: (
         color: map-get($palette, kicker);
     }
 
-    &.fc-item--type-immersive.fc-item--has-boosted-title:not(.fc-item--pillar-special-report) .fc-item__standfirst {
+    &.fc-item--type-immersive.fc-item--has-boosted-title:not(.fc-item--pillar-special-report):not(.fc-item--pillar-special-report-alt) .fc-item__standfirst {
```

Comment:
I tried `:not(.fc-item--pillar-special-report, .fc-item--pillar-special-report-alt)` but didn't work!